site/contents/write_ups/2022/01-06-22_writeup.md

```diff
@@ -0,0 +1,94 @@
+
+## Is the most participation always the best?
+
+We were of the opinion that most partcipation is *not* always the best. 
+
+You need to commit to a certain level of maintenance for data to be worked with. If this needs to be run by the participants, this can really bog things down. (This will be evident to anyone who has ever tried to organise a group holiday).
+
+There's a difficulty of presenting it as a hierarchy, implying one is better and one is worse. 
+
+We did appreciate that this report seemed to have a section on how to make each action meaningful. That's not often addressed, usually it's just awareness.
+
+When thinking about large data sets for training ML models (e.g. image recognition), how do you handle participation levels/consent? If you have 100k pictures of people trying to guess your hair colour, do you need to sign off on every new study/model or give blanket consent?
+
+Can you give consent for something you didn't know was going to happen? Often large data sets are not curated for a particular purpose so you just have to trust that the people who publish them collate the data in an ethical way. The scale of these datasets makes this pretty much impossible. Caution has already been warned on handling such [large datasets](https://arxiv.org/abs/2110.01963).
+
+Even when you do open source, public research, it'll be difficult sometimes to explain every single project to everyone. Even when you can't get informed consent (be it because people are bad at stats or because your subjects can't consent, like plants), you can still have legal frameworks to ensure the data is ethically used and maintained.
+
+Because humans are sentient, it opens you up to negotiations. With data about plants for example, there is no negotiation with the objects. You make a lot of decisions on their "behalf".
+
+"If you did understand, you'd love it and think it was super ethical I swear." 
+
+"But I don't understand it." 
+
+"But if you did..."
+
+There has been some interesting research on how privacy statements are worded got you more access (even if they were legally equivalent, but with behaivoural nudges). Why would anyone ever do anything bad with this data? E.g. adobe releasing video editing software that enabled deep fakes.
+
+Is it always good to have participation? Maybe it's good to mandate some level of participation, even if it's not the most stringent level. Thinking about people who are underrepresented for example.
+
+## What do you see as being barriers to participatory data stewardship, and how could these realistically be overcome?
+
+This effort can be demanding on researchers. Perhaps there should be "centralised" institutional support, creating "pools" of relevant participants to reduce duplication of effort. There is a lack of funding to practically implement this framework.
```

Comment:
```suggestion
This effort can be demanding on researchers, and if the level of participation is high it can be demanding on participants too. Perhaps there should be "centralised" institutional support, creating "pools" of relevant participants to reduce duplication of effort. There is a lack of funding to practically implement this framework.
```